include/ethash/keccak.h

```diff
@@ -19,6 +19,11 @@
 extern "C" {
 #endif
 
+typedef void (*ethash_keccakf1600_func)(uint64_t[25]);
+
+/// The pointer to the Keccak-f[1600] function implementation.
+extern ethash_keccakf1600_func ethash_keccakf1600;
```

Comment:
Maybe instead of exposing global variable it would look simpler to have another wrapper function like `ethash_keccakf1600_best` or something

Reply:
Or just `ethash_keccakf1600` that would call `ethash_keccakf1600_best` function pointer inside

Reply:
I considered this, but that mean call + indirect call instead of just indirect call.
Anyway, don't want to expose `keccakf1600` any more. It is convenient for benchmarking only so I will either expose these under some macro `KECCAK_EXPORT_INTERNALS` or will just use `keccak256(nullptr, 0)`. I'm not sure why I exposed these in the first place.

But I don't want to do it here.